projects/packages/forms/src/contact-form/class-jetpack-form.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Jetpack_Form class.
+ *
+ * @package automattic/jetpack-forms
+ */
+
+namespace Automattic\Jetpack\Forms\ContactForm;
+
+/**
+ * Jetpack_Form helper class for the Jetpack Form Custom Post Type.
+ */
+class Jetpack_Form {
+
+	const POST_TYPE = 'jetpack-form';
```

Comment:
Core uses underscores and, looking around the JP codebase, we mostly use underscores for CPTs. Worth switching for consistency? Technically no difference tho.